test/integration/webhooks/leaderworkerset_test.go

```diff
@@ -111,10 +111,15 @@ var _ = ginkgo.Describe("leaderworkerset defaulting, creation and update", func(
 		}),
 		ginkgo.Entry("apply default rollout strategy", &testDefaultingCase{
 			makeLeaderWorkerSet: func(ns *corev1.Namespace) *testutils.LeaderWorkerSetWrapper {
-				return testutils.BuildLeaderWorkerSet(ns.Name).RolloutStrategy(leaderworkerset.RolloutStrategy{}) // unset rollout strategy
```

Comment:
Why change this? BuilderLeaderWorkerSet also set the rolloutStrategy, so what you changed here will not validate the defaulter for rollout strategy.